src/context/MessengerContext.js

```diff
@@ -66,6 +79,7 @@ class MessengerContext extends Context implements PlatformContext {
     return this._client.sendText(this._session.user.id, text, {
       messaging_type: messageType,
       ...options,
+      access_token: this._customAccessToken,
```

Comment:
should we avoid adding this key (`access_token`) when `_customAccessToken` is not present?

Reply:
prevent something like `{ ...oprions, access_token: undefined }`

Reply:
maybe we should use messaging-api to handle this part?